src/xcode/ENA/ENA/Source/Scenes/RecycleBin/Cells/RecycleBinItemCellModel.swift

```diff
@@ -30,7 +32,9 @@ class RecycleBinItemCellModel {
 		switch recycleBinItem.item {
 		case .certificate(let certificate):
 			return certificate.name.fullName
-		case .coronaTest:
+		case .userCoronaTest:
```

Comment:
```suggestion
		case .userCoronaTest, .familyMemberCoronaTest:
```

Reply:
Just added the name to be displayed here for family member tests, so the separate case makes sense now